README.md

```diff
@@ -2,29 +2,7 @@
 
 PyNear is a python library, internally built in C++, for efficient KNN search using metric distance function such as L2 distance (see VPTreeL2Index) or Hamming distances (VPTreeBinaryIndex) as well as other indices. It uses AVX2 instructions to optimize distance functions so to improve search performance.
 
-## How VP-Trees work
```

Comment:
Note: moved this vp tree explanation to a separate file in docs as it was polutting front page a big and now is not our main focus to provide only vp trees.